### DISCUSSION TITLE
mlc-ai-nightly-cu110 package installation issue for EP7

### DISCUSSION THREAD

**Discussion by voyagerdd** [mlc-ai/mlc-zh#98] [Q&A]
Got the following error while installing mlc-ai-nightly-cu110 package:
ERROR: Could not find a version that satisfies the requirement mlc-ai-nightly (from versions: none)
ERROR: No matching distribution found for mlc-ai-nightly

$ nvidia-smi 
Sun Aug  7 11:38:37 2022       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 465.19.01    Driver Version: 465.19.01    CUDA Version: 11.3     |

Is this because the CUDA version I'm running is not supported by the package? If so, could a package supporting the CUDA version be provided? Thanks

**Comment by Hzfengsy** [comment]
Could you provide your OS type and python version? We only build `cu110` package for Linux with Python 3.7 and 3.8

BTW, it's `zh` site. Please feel free to speak Chinese:) or you can write in the [mlc-en](https://github.com/mlc-ai/mlc-en/discussions/) discussion.

**Reply by voyagerdd** [reply]
感谢回复！OS：Ubuntu 20.04.2 LTS; Python 3.8.5. 请问cu110 package支持这个组合吗？如果不支持有什么办法解决吗？谢谢

**Reply by Hzfengsy** [reply]
我测试了Python3.8 + 20.04是可以安装的，可以检查一下是否是网络问题（可能需要科学上网）

**Reply by voyagerdd** [reply]
我是科学上网后先把mlc_ai_nightly_cu110-0.9.dev1956+ge3f218d71-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl下载到本地，然后执行下面的本地安装命令。下载的文件大小是正常的200+M
$ python3 -m  pip install mlc-ai-nightly -f file:///path/to/mlc_ai_nightly_cu110-0.9.dev1956+ge3f218d71-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
Looking in links: file:///path/to/mlc_ai_nightly_cu110-0.9.dev1956+ge3f218d71-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
ERROR: Could not find a version that satisfies the requirement mlc-ai-nightly (from versions: none)
ERROR: No matching distribution found for mlc-ai-nightly
报错信息如上，请问有什么建议的解决方法吗？谢谢。或者可以帮忙把需要安装的包在对国内友好的环境也放一份吗？

**Reply by Hzfengsy** [reply]
尝试一下 `pip install path/to/your/wheel`。

国内环境比较麻烦，需要网络资源、域名备案等一系列问题，目前课程官方没有相应的资源来支持，非常抱歉。

如果社区有人能够帮忙提供相关，我们支持把资源镜像到国内

**Reply by voyagerdd** [reply]
Great! it works. really appreciate your help!

**Reply by yinghuoai** [reply]
> 我的解决方法如下，希望可以帮到你。

我跳到 [https://mlc.ai/wheels](url) 页面，复制了对应版本的whl文件的链接，然后python命令安装。
环境是在colab下：
`
! python3 -m pip install https://github.com/mlc-ai/utils/releases/download/v0.9.dev0/mlc_ai_nightly_cu113-0.9.dev2968%2Bgccfaa77f9-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl -f https://mlc.ai/wheels
`
![uTools_1672739971951](https://user-images.githubusercontent.com/29194219/210335337-f4e962c9-e617-4d1a-975b-300a1128277b.png)

**Reply by KevinZeng08** [reply]
链接更新了，我用的：
`!python3 -m pip install https://github.com/mlc-ai/package/releases/download/v0.9.dev0/mlc_ai_nightly_cu116-0.12.dev1310-cp310-cp310-manylinux_2_28_x86_64.whl -f https://mlc.ai/wheels`

**Reply by BestDreamy** [reply]
链接更新了：
!python3 -m pip install https://github.com/mlc-ai/package/releases/download/v0.9.dev0/mlc_ai_nightly_cpu-0.20.dev562-py3-none-manylinux_2_28_x86_64.whl -f https://mlc.ai/wheels
